### PR TITLE
Added Pylint for additional code quality checks

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

**Comment by richardbarran** [comment]
Hi Clinton - if you can rebase this branch on `master` I'd be happy to have a go at reviewing it.

File: sortedm2m_tests/test_field.py [hunk 1]
```diff
@@ -261,15 +263,4 @@ def test_self_adding(self):
         s1.me.add(s3)
         s1.me.add(s4, s2)
 
-        self.assertEqual(list(s1.me.all()), [s3,s4,s2])
-
-
-class TestDjangoManyToManyFieldNotAvailableThroughSortedM2M(TestCase):
```
Comment (richardbarran) [inline sortedm2m_tests/test_field.py]: A quick question: why remove this test?

**Comment by richardbarran** [comment]
Maybe add to the README a few lines on how to run pylint for this project?

File: sortedm2m_tests/test_field.py (hunk 1 above)
Comment (richardbarran) [reply sortedm2m_tests/test_field.py]: Ok, sounds good to me!

**Comment by clintonb** [comment]
@richardbarran 

1. Updated.
2. Yes, it is possible, but I don't think the validation should be ignored globally. We have some cases where we need to legitimately disable the validation. That is not always the case.

**Comment by richardbarran** [comment]
Nice work!

**Comment by nemesifier** [comment]
Thanks :+1:

**Pull request merged**